Capstone/Detail View/DetailsViewController.m

```diff
@@ -58,6 +58,17 @@ - (void)handleDoubleTap:(UITapGestureRecognizer *)sender {
     [self performSegueWithIdentifier:@"moreDetails" sender:self];
 }
 
+- (IBAction)didTapDirectionsToCollege:(id)sender {
+    CLLocationCoordinate2D coordinate = [self getLocation];
+    CLLocationCoordinate2D start = {coordinate.latitude, coordinate.longitude};
+    CLLocationCoordinate2D destination = {[self.college.lat doubleValue], [self.college.longtuide doubleValue]};
+
+    NSString *googleMapsURLString = [NSString stringWithFormat:@"http://maps.google.com/?saddr=%1.6f,%1.6f&daddr=%1.6f,%1.6f",
+                                     start.latitude, start.longitude, destination.latitude, destination.longitude];
```

Comment:
is start.latitude and start.longitude different from coordinate.latitude and coordinate.longitude?

Reply:
Yes. So, the starting location is the user's location(longitude and latitude) with the end location being the colleges coordinates(longitude and latitude).

Reply:
How about we use rename `coordinate` to `startCoordinate`, and use `coordinate.longitude` & `coordinate.latitude` instead of a new variable?